docs/user-guide/aroma/finalizing-setup.md

```diff
@@ -2,31 +2,41 @@
 
-1. Take the SD Card out of your computer and plug it into your Wii U console.
-1. Start the EnvironmentLoader.
-    - If you're autobooting into Tiramisu, this can be done by powering on your console and holding `X`.
-1. Highlight the entry called `aroma` using the D-Pad.  
-    - To make the console autoboot Aroma press `Y`.
-1. Launch Aroma by pressing `A`.
-    - If you get a red warning screen telling you that updates aren't blocked properly, you might want to block updates by following [this guide](../block-updates).
-1. On the Boot Selector, the `Wii U Menu` should already be selected, press Y to set this to your default autobooting option, then press A to launch into the Wii U Menu.
-    - To open the Boot Selector in the future, you have hold START (+) while launching Aroma.
+### Setting up PayloadLoader, Environment Loader and Aroma
+
+1. Turn on your Wii U.
+    - The Environment Loader should show up.
+1. Using the D-Pad, navigate to `aroma` and press Y to set this to your default environment, then press A to launch into Aroma.
+    - You might get a red warning screen telling you that updates aren't blocked properly. Press A to continue anyway. We're going to block updates in the "Blocking Updates" section below.
+    - To open the Environment Loader in the future, you have to hold X while your Wii U is booting up.
+1. On the Aroma Boot Selector, the `Wii U Menu` should already be selected, press Y to set this to your default autobooting option, then press A to launch into the Wii U Menu.
+    - To open the Aroma Boot Selector in the future, you have hold START (+) while your Wii U is booting up.
+
+### Blocking Updates
+While Aroma's PayloadLoader already has built-in update blocking functionality, it is recommended to delete the update folder to effectively block system updates.
+If you get a red warning screen while booting into Tiramisu, the update folder still exists and it is recommended to delete it using [this guide](../block-updates).
+
+### Additional Homebrew Apps
+
+- **SaveMii Mod WUT Port** is a homebrew app that lets you manage your Wii U and vWii save data.
+- **Bloopair** is an Aroma module that lets you wirelessly connect most popular Bluetooth capable controllers. See [this page](https://gbatemp.net/threads/bloopair-connect-controllers-from-other-consoles-natively.594289/) for more details.
+- The **Homebrew Appstore** lets you browse and download homebrew apps directly from your Wii U.
 
-You are now running Aroma. Any apps you have on your SD Card will show up on the Wii U menu.
+### Additional Homebrew Apps - What You Need
 
-?> **Aroma does not support the Homebrew Launcher**  
-Launch apps from the Wii U menu using the `homebrew_on_menu` plugin.
```

Comment:
It might still be worth explaining that apps on the SD Card appear on the Wii U Menu, even though the Homebrew Launcher part is no longer relevant.

Reply:
That is true